nexus/src/external_api/http_entrypoints.rs

```diff
@@ -7372,7 +7378,7 @@ async fn user_list(
     let handler = async {
         let opctx = OpContext::for_external_api(&rqctx).await?;
         let users = nexus
-            .silo_users_list_current(&opctx, &pagparams)
+            .silo_users_list_current_by_id(&opctx, &pagparams)
```

Comment:
I could make them both use the same method like you did here if the group filter query param is purely additive.
https://github.com/oxidecomputer/omicron/pull/2420/files#diff-7429b402f97c8a901b88c3599e8505de4afae78092b5f27e4542bc5818125f17R5794-R5798